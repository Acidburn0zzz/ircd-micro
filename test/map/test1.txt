+a=A
+b=B
+c=C
=a
=b
=c
=d
-b
=a
=b
=c
=d
q
