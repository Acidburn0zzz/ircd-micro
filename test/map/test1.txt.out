A
B
C

B
A

C

bye
